embulk-output-redshift/src/main/java/org/embulk/output/redshift/RedshiftOutputConnection.java

```diff
@@ -38,45 +36,27 @@ public void dropTableIfExists(TableIdentifier table) throws SQLException
             String sql = String.format("DROP TABLE IF EXISTS %s", quoteTableIdentifier(table));
             executeUpdate(stmt, sql);
             commitIfNecessary(connection);
-        } catch (SQLException ex) {
-            // ignore errors.
-            // TODO here should ignore only 'table "XXX" does not exist' errors.
-            SQLException ignored = safeRollback(connection, ex);
         } finally {
             stmt.close();
         }
     }
 
-    // Redshift does not support DROP TABLE IF EXISTS.
-    // Dropping part runs DROP TABLE and ignores errors.
     @Override
     public void replaceTable(TableIdentifier fromTable, JdbcSchema schema, TableIdentifier toTable, Optional<String> additionalSql) throws SQLException
```

Comment:
We can remove this method because the super class (AbstractJdbcOutputPlugin) does same thing.
`DROP` and `ALTER TABLE` were already in one transation without "BEGIN TRANSACTION" because autoCommit is false.